openassessment/xblock/studio_mixin.py

```diff
@@ -379,19 +347,13 @@ def _editor_assessments_order_context(self):
         # since the user last saved their ordering.
         effective_order = copy.deepcopy(DEFAULT_EDITOR_ASSESSMENTS_ORDER)
 
-        # If the problem already contains example-based assessment
-        # then allow the editor to display example-based assessments,
-        # which is not included in the default
-        enabled_assessments = [asmnt['name'] for asmnt in self.valid_assessments]
```

Comment:
From my database poking, I don't believe there are any. I'll verify that, and set up such a problem locally to ensure we can handle that situation correctly.